src/feature/MovieBrowser/Movies/List/PopularMovies/Data/Rating/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```

Comment:
![image](https://user-images.githubusercontent.com/123471023/232243467-6cf4e1f5-5da4-4d61-9d2c-bb3c14c9c4da.png)
Now the votes & rating are working!